docs/environments.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Environments
+---
+
+# Environments
+
+## Descriptions
+
+Event Bus [Kafka](https://kafka.apache.org/) and applications are deployed into several environments for different purposes. The environments are described below.
+
+- **`Dev`** is an unstable environment intended for Event Bus team's development work. No customer applications will be deployed here.
```

Comment:
Can we change the language here from "deployed here" to "connected here" -- we're not going to be deploying any consumer applications in our environments.

Reply:
I've changed "deployed" to "connected".